x-pack/test/security_solution_cypress/cypress/tasks/select_all_spaces.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SPACE_SELECTOR_COMBO_BOX } from '../screens/custom_roles/assign_to_space_flyout';
+
+export const selectAllSpaces = (): void => {
+  cy.get('[data-test-subj="addSpacePrivilegeButton"]').click();
```

Comment:
NIT: Use the constant you created in the screens folder.

Reply:
done 894fdfb79f04458aa55138df0fab3bfb00a3c540